github/ci/prow/files/config.yaml

```diff
@@ -183,6 +183,8 @@ branch-protection:
           branches:
             master:
               protect: true
+              required_status_checks:
```

Comment:
`dco` seems to be a required but it does not appear here.
Generally speaking, based on the docs, at least one required check must exist... hope the `dco` one is enough.

Reply:
Hm, maybe we should also make the check-provision make required?